src/main/java/dataMgmt/SessionStorage.java

```diff
@@ -40,6 +68,57 @@ public void setAuthenticationToken(String authenticationToken) {
     }
 
     /**
+     * Get the current user's loaded projects.
+     * @return projects list
+     */
+    public List<Project> getProjects() {
+        List<Project> projects = new ArrayList<>(this.projects.values());
+        Collections.sort(projects, (o1, o2) -> {
+            if (o1.getName() == null) {
```

Comment:
In theory it can't ever not have a name, but usually in compareTo functions you check that case anyway when comparing Objects.